orchagent/observer.h

```diff
@@ -2,12 +2,14 @@
 #define SWSS_OBSERVER_H
 
 #include <list>
+#include <algorithm>
 
 using namespace std;
 using namespace swss;
 
 enum SubjectType
 {
+    SUBJECT_TYPE_ALL_CHANGES,
```

Comment:
Where is this used? If not, please remove

Reply:
@prsunny :Thanks, the code used to use was part of optimization for the subject calss, which is no more part of the PR. Removed the attribute.